index.json

```diff
@@ -50,6 +50,14 @@
   },
   "faraday": {
   },
+  "globalid": {
+    "dependencies": [
+      "activerecord"
```

Comment:
We don't actually need this because missing constant errors are turned off on CI.

Reply:
TIL - removed it! thanks :)